src/content/resources/concepts/layer/assessment/poam/_index.md

```diff
@@ -13,7 +13,7 @@ aliases:
 
 | POA&M Schema | POA&M Converters | Reference
 |:--- |:--- |:--- |
-| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_poam_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_poam_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/json#converting-oscal-xml-content-to-json)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/plan-of-action-and-milestones/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/plan-of-action-and-milestones/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/plan-of-action-and-milestones/json-index/) |
+| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_poam_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_poam_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/blob/main/build/README.md#converters)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/plan-of-action-and-milestones/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/plan-of-action-and-milestones/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/plan-of-action-and-milestones/json-index/) |
 | [XML Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_poam_schema.xsd) | [JSON to XML Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_poam_json-to-xml-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/plan-of-action-and-milestones/xml-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/plan-of-action-and-milestones/xml-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/plan-of-action-and-milestones/xml-index/) |
```

Comment:
https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml is broken and needs to be replaced.

Reply:
It seems I must have done a find/replace incorrectly. Sorry I have not finished touch up of this like I said I would do during sprint review today. I am going to come back to this later tonight or tomorrow. Thanks for catching these.